### ISSUE TITLE
Bug in PDF Exports causing wide gaps between letters

### ISSUE THREAD

**Issue by juan-rivera-2020** [MicroHealthLLC/mPATH#2628]
Recently reported by field user:  PDF exports Last Update column prints with wide spaces between letters.  This case was caused by copy and pasting text from another source which introduced the characters that should have been "sanitized" prior to saving.   After researching this issue, we discovered that the export library used in mPATH (jsPDF) currently [does not _properly_ support](https://github.com/MrRio/jsPDF/issues/2640) UTF-8,  a "variable-width character encoding used for electronic communication". Essentially, UTF-8 would have sanitized these special characters.    Since jsPDF is a popular and commonly used library for document export functions, we decided to fix this bug instead of replacing the library.  

![image](https://user-images.githubusercontent.com/72885316/118719922-04c8b680-b7f7-11eb-8d31-f92941e8de7a.png)

**Comment by juan-rivera-2020** [comment]
This bug was fixed at the create/edit level.  Therefore, any new text being introduced will be sanitized prior to saving.  Additionally, previous "Last Updates" with this issue were handled with regex applied directly on datatable such that the table print out will correctly display the data.  Screenshot after fixing bug: 
![image](https://user-images.githubusercontent.com/72885316/118720782-08a90880-b7f8-11eb-8ea5-b5293220018c.png)

**Comment by lingzhang1** [comment]
Hi @juan-rivera-2020 ,

I got same issue when using jsPDF. May I ask how did you fix the bug?

**Comment by juan-rivera-2020** [comment]
@lingzhang1 Ni hao.  I fixed it with regex.   Specifically, I used this regex:  `replace(/[^ -~]/g,'')`  In my case, there were two different areas in my app that required this fix.  In the frontend, I added this : `exampleValue.replace(/[^ -~]/g,'')` to the value I was rendering in my datatable .  The other place where I used it was at the create/update level so I can "sanitize" the code before it gets saved.
